app/views/idv/shared/_document_capture.html.erb

```diff
@@ -142,7 +142,13 @@
 
   <p class='margin-top-4 margin-bottom-0'><%= t('doc_auth.info.document_capture_upload_image') %></p>
 </div>
-<%= render 'idv/doc_auth/start_over_or_cancel' %>
+
+<%= render 'idv/doc_auth/start_over' %>
+<div class='margin-top-2 padding-top-1 border-top'>
+  <%= button_to(cancel_link_text, idv_capture_doc_step_url(step: :cancel), method: :put,
+                class: 'btn btn-link') %>
```

Comment:
This partial is shared by both the `doc_auth` and `capture_doc` flows, but if I'm following correctly, these changes should only apply to the `capture_doc` flow? Testing the normal desktop flow, clicking "Cancel" now brings me back to the account page with an error:

![image](https://user-images.githubusercontent.com/1779930/110323615-d488be80-7fe2-11eb-91cb-652d7b3ea2b2.png)

Reply:
Great catch!